app/api/report.py

```diff
@@ -210,6 +212,47 @@ def week_checkin_reports():
     return jsonify(docs), 200
 
     
+@bp.route('/revoked_checkin', methods=["GET"])
+@jwt_required
+def revoke_checkin_reports():
+    current_user = get_current_user()
+    date_t=current_user["revoke"]
+    today = date_t
+    last_sunday = today - datetime.timedelta(days=(today.weekday() + 1))
+    last_monday = today - datetime.timedelta(days=(today.weekday() + 8))
+    current_user = get_current_user()
+    docs = mongo.db.reports.find({
+        "user": str(current_user["_id"]),
+        "type": "daily",

```

Comment:
why are we checking daily report.  this has nothing to do with daily reports

Reply:
this is because we are showing last to last checkins on weekly submit page.